sdk/keyvault/azure-keyvault-administration/tests/recordings/test_access_control.pyTestAccessControltest_role_assignment[7.2].json

```diff
@@ -1,7 +1,7 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://managedhsmvaultname.vault.azure.net/providers/Microsoft.Authorization/roleDefinitions?api-version=7.2",
+      "RequestUri": "https://managedhsmvaultname.managedhsm.azure.net/providers/Microsoft.Authorization/roleDefinitions?api-version=7.2",
```

Comment:
Given how much size recordings contribute to the repos, are these necessary? No difference in recordings would be significant here.

Reply:
This is unfortunately necessary because of the validation check. I suppose we could alternatively initialize our client with `verify_challenge_resource=False`, but otherwise our MHSM tests fail in playback because our playback URL was using the wrong URL suffix. Great for validating that the check works, but less than ideal for backwards compatibility

Reply:
I wouldn't stress about it, then. I just thought maybe it was happenstance.